src/database/config/config.js

```diff
@@ -13,10 +13,7 @@ module.exports = {
     },
   },
   test: {
-    url: TEST_DATABASE_URL,
-    dialectOptions: {
-      ssl,
-    },
+    url: `postgresql://postgres:${process.env.POSTGRES_PASSWORD}@localhost:5432/postgres`,
```

Comment:
```suggestion
    url: TEST_DATABASE_URL`,
```
can put the url in the environment variable instead

Reply:
Resolved.